pages/shared/syncHelpers.js

```diff
@@ -303,12 +310,164 @@ module.exports.gitStatus = function (locals, callback) {
   });
 };
 
+function locateImage(image, callback) {
```

Comment:
All of the following code was only ever used in this file. Rather than trying to convert it to TypeScript and modernize it, I opted to just inline it here.

Reply:
Reasonable.